snap/snapcraft.yaml

```diff
@@ -1,55 +1,74 @@
 name: srsran
 base: core20
-version: '22.04.1'
+version: '20.10'
 summary: srsRAN is a 4G/5G software radio suite developed by SRS.
 description: srsRAN is a 4G/5G software radio suite developed by SRS.
-
 grade: devel
 confinement: devmode
-
 apps:
+  srsenb:
+    command: usr/bin/srsenb
+    plugs:
+      - home
+      - network
+      - network-bind
+      - network-control
+      - log-observe
+      - process-control
+  srsue:
+    command: usr/bin/srsue
+    plugs:
+      - home
+      - network
+      - network-bind
+      - network-control
   srsepc:
-    command: usr/local/bin/srsepc
+    command: usr/bin/srsepc
     plugs:
       - home
       - network
       - network-bind
       - network-control
 
-parts:
-
-  libzmq:
-    plugin: autotools
-    source: https://github.com/zeromq/libzmq.git
-    source-type: git
-    source-tag: v4.3.4
-
-  czmq:
-    after:
-      - libzmq
-    plugin: autotools
-    source: https://github.com/zeromq/czmq.git
-    source-type: git
-    source-tag: v4.2.1
-
-  srsran:
+layout:
+  /usr/share/srslte:
+    bind: $SNAP/usr/share/srslte
+  /etc/srslte:
+    bind: $SNAP/usr/share/srslte
+parts: 
+  srslte:
     plugin: cmake
-    after:
-      - czmq
-    source: https://github.com/srsRAN/srsRAN.git
+    cmake-parameters:
+      - -DCMAKE_BUILD_TYPE=RelWithDebInfo
+      - -DENABLE_ASAN=True
+      - -DCMAKE_INSTALL_PREFIX=/usr
+    source: https://github.com/srsLTE/srsLTE.git
     source-type: git
-    source-tag: release_22_04_1
+    source-tag: release_20_10_1
     build-packages:
-      - build-essential
-      - cmake
-      - libfftw3-dev
+      - libzmq3-dev
       - libmbedtls-dev
       - libboost-program-options-dev
-      - libconfig++-dev
       - libsctp-dev
+      - libconfig++-dev
+      - libfftw3-dev
     stage-packages:
+      - libzmq5
       - libboost-program-options1.71.0
-      - libconfig++9v5
+      - libsctp1
       - libfftw3-single3
       - libmbedcrypto3
-      - libsctp1
+      - libconfig++9v5
+      - libasan5
+    prime:
+      - -usr/include
+    override-prime: |
+      snapcraftctl prime
+      mv ./usr/share/srslte/ue.conf.example ./usr/share/srslte/ue.conf
```

Comment:
Can't we use the `organize` keyword instead of all those `mv`'s?

For reference:
```
organize
A map of files to rename.
Type: dict
In the key/value pair, the key represents the path of a file inside the part and the value represents how the file is going to be staged.
```

Reply:
✅